Modules/Tooltips/Tooltip.lua

```diff
@@ -69,8 +69,24 @@ function QuestieTooltips:RemoveQuest(questId)
         return
     end
 
-    for _, key in pairs(QuestieTooltips.lookupKeysByQuestId[questId]) do
-        QuestieTooltips.lookupByKey[key] = nil
+    for _, key in pairs(QuestieTooltips.lookupKeysByQuestId[questId] or {}) do
+        --Count to see if we should remove the main object
+        local totalCount = 0
+        local totalRemoved = 0
+        for _, tooltipData in pairs(QuestieTooltips.lookupByKey[key] or {}) do
```

Comment:
I removed the `or {}` in this line and the loop above, because if this erros out it means we have a data leak, doesn't it? It also errored out for me, didn't find the cause for it though.

Reply:
The debug stack says it is `RemoveQuest` which is called "every now and then" which will then lead to `QuestieTooltips.lookupByKey[key]` being nil. When you accept a quest and then remove it, it always errors so it should be easy to investigate.

Maybe it is not really worth investigation though, but I think there is a data leak somewhere.